scripts/create-user.sh

```diff
@@ -18,7 +18,7 @@ MESSAGE='Hello,
 Your account for GOV.UK PaaS is ready:
 
  - username: ${EMAIL}
- - organisation: ${ORG}
+${ORG:+ - organisation: ${ORG}}
```

Comment:
Minor: is there a not-terrible way to omit the extra newline that this generates?